homeassistant/components/enphase_envoy/config_flow.py

```diff
@@ -89,24 +89,41 @@ async def async_step_zeroconf(
         self, discovery_info: zeroconf.ZeroconfServiceInfo
     ) -> ConfigFlowResult:
         """Handle a flow initialized by zeroconf discovery."""
+        current_hosts = self._async_current_hosts()
```

Comment:
We could guard this with [isEnabledFor()](https://docs.python.org/3/library/logging.html#logging.Logger.isEnabledFor) since its only used if debug logging is enabled and will be throw away otherwise.

https://docs.python.org/3/howto/logging.html#optimization

Reply:
While we're at it, added.

Reply:
I added debug to one of the tests and observed the debug line showing up in local pytest -vv -s. But codecov here doesn't show the coverage. I'm porbably doing something the wrong way.

Reply:
If you add `caplog` and check the log message it will likely cover it

Reply:
That worked, thanks.